packages/map-template/src/components/Map/MapboxMap/MapboxMap.jsx

```diff
@@ -48,6 +51,11 @@ function MapboxMap({ onMapView, onPositionControl }) {
             mapViewOptions.mapsIndoorsTransitionLevel = parseInt(miTransitionLevel);
         }
 
+        // If showRoadNameLabels is not null or undefined.
```

Comment:
Yes, that's what the condition says 😄  Maybe change the comment to describe what happens if the condition is met.

Reply:
✅